src/twisted/python/test/test_release.py

```diff
@@ -356,12 +357,38 @@ def test_replaceInFile(self):
 
 
 
+def doNotFailOnNetworkError(func):
+    """
+    A decorator which makes APIBuilder tests not fail because of intermittent
+    network failures -- mamely, APIBuilder being unable to get the "object
+    inventory" of other projects.
+
+    @param func: The function to decorate.
+
+    @return: A decorated function which won't fail if the object inventory
+        fetching fails.
+    """
+    @functools.wraps(func)
+    def _(*a, **kw):
```

Comment:
Can this be `wrapper(*args, **kwargs)`?
`_` is OK for a symbol that's ignored, but this is actually used again later, so ew.